Fix rocDecode to use EasyBuild ROCm-LLVM compiler paths

In EasyBuild, compilers are in EBROOTROCMMINLLVM/bin not ROCM_PATH/lib/llvm/bin

Author: sreenivasamurthy kolam <sreenivasamurthy.kolam@amd.com>

--- CMakeLists.txt.orig	2026-09-24 15:30:38.577447431 +0530
+++ rocDecode-rocm-7.2.3/CMakeLists.txt	2026-09-24 15:30:46.041468196 +0530
@@ -42,8 +42,8 @@
 set(CMAKE_CXX_STANDARD_REQUIRED On)
 set(CMAKE_CXX_EXTENSIONS ON)
 if (NOT DEFINED CMAKE_CXX_COMPILER)
-  set(CMAKE_C_COMPILER ${ROCM_PATH}/lib/llvm/bin/amdclang)
-  set(CMAKE_CXX_COMPILER ${ROCM_PATH}/lib/llvm/bin/amdclang++)
+  set(CMAKE_C_COMPILER $ENV{EBROOTROCMMINLLVM}/bin/amdclang)
+  set(CMAKE_CXX_COMPILER $ENV{EBROOTROCMMINLLVM}/bin/amdclang++)
 endif()
 
 # rocdecode Version
